### ISSUE TITLE
Dependencies that are imported with path, git,.. are not supported yet

### ISSUE THREAD

**Issue by vanlooverenkoen** [icapps/flutter-icapps-license#17]
Currently it is not possible to add path, git dependencies.

You could use #13 or #3 for that. But the error will still be shown for now

In the example project we get this.
```
The license for icapps_license could not be fetched automaticly. icapps_license should define a static license or url in the pubspec.yaml (https://pub.dev/packages/icapps_license)
```

**Comment by vanlooverenkoen** [comment]
This will be fixed in the refactor

**Comment by vanlooverenkoen** [comment]
This is included in our new package license_generator. It is released with version: 1.0.0
